### PR TITLE
New indicators layouts - part 2

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
relates to: https://github.com/demisto/etc/issues/23383

## Screenshots
image
image

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [x] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/CommonTypes/Layouts/layout-indicatorsQuickView-account.json [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+                ],
+                "type": "custom"
+            }
+        ],
+        "typeId": "accountRep",
+        "version": -1
+    },
+    "typeId": "accountRep",
+    "version": -1,
+    "fromVersion": "5.5.0",
```
Comment (Itay4) [inline Packs/CommonTypes/Layouts/layout-indicatorsQuickView-account.json:198]: remove this file please

**Comment by Shellyber** [comment]
@kirbles19 Please review RN

File: Packs/CommonTypes/Layouts/layout-indicatorsDetails-Account.json [hunk 2]
```diff
@@ -8,10 +8,10 @@
 		"modified": "2020-03-26T11:59:52.044505+02:00",
 		"name": "",
 		"system": false,
-		"tabs": [
+				"tabs": [
```
Comment (Itay4) [inline Packs/CommonTypes/Layouts/layout-indicatorsDetails-Account.json:11]: fix indent here please

File: Packs/CommonTypes/Layouts/layoutscontainer-registryKey.json [hunk 3]
```diff
@@ -7,306 +7,683 @@
...
+						"x": 2,
+						"y": 0
+					}
+				],
+				"type": "custom"
+			}
+		]
+    },
+    "indicatorsQuickView": {
+		"tabs": [
```
Comment (Itay4) [inline Packs/CommonTypes/Layouts/layoutscontainer-registryKey.json:542]: fix indent here please

File: Packs/CommonTypes/Layouts/layoutscontainer-registryKey.json [hunk 4]
```diff
@@ -7,306 +7,683 @@
...
+						"static": false,
+						"type": "customDetails",
+						"w": 1,
+						"x": 2,
+						"y": 0
+					}
+				],
+				"type": "custom"
+			}
+		]
```
Comment (Itay4) [inline Packs/CommonTypes/Layouts/layoutscontainer-registryKey.json:539]: fix indent here please

File: Packs/CommonTypes/Layouts/layoutscontainer-registryKey.json [hunk 5]
```diff
@@ -7,306 +7,683 @@
...
+						"name": "Registry Key details",
+						"static": false,
+						"w": 1,
+						"x": 0,
+						"y": 4
+					}
+				],
+				"type": "custom"
+			}
+		]
```
Comment (Itay4) [inline Packs/CommonTypes/Layouts/layoutscontainer-registryKey.json:687]: fix indent here please

File: Packs/CommonTypes/Layouts/layoutscontainer-registryKey.json (hunk 3 above)
Comment (Shellyber) [reply Packs/CommonTypes/Layouts/layoutscontainer-registryKey.json:542]: image
Looks good in my env.

**Comment by Shellyber** [comment]
@Itay4 Indents looks ok in my env

File: Packs/CommonTypes/Layouts/layoutscontainer-registryKey.json (hunk 4 above)
Comment (Shellyber) [reply Packs/CommonTypes/Layouts/layoutscontainer-registryKey.json:539]: image

**Comment by Itay4** [comment]
image

image

image

image

**Comment by ShirleyDenkberg** [comment]
@Shellyber Done

File: Packs/CommonTypes/Layouts/layoutscontainer-Account.json [hunk 6]
```diff
@@ -1,376 +1,779 @@
 {
-	"id": "accountRep",
-	"group": "indicator",
-	"name": "Account Indicator",
-	"description": "",
-	"version": -1,
-	"fromVersion": "6.0.0",
+    "id": "accountRep",
+    "group": "indicator",
+    "name": "Account layout by XSOAR",
```
Comment (Itay4) [inline Packs/CommonTypes/Layouts/layoutscontainer-Account.json:4]: why is this the name?

Comment (Shellyber) [reply Packs/CommonTypes/Layouts/layoutscontainer-Account.json:4]: I was asked to change all names to 6.0 layouts

**Pull request merged**